specification/search/data-plane/Microsoft.Azure.Search.Service/track1/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5122,16 +5353,6 @@
       },
       "description": "A string indicating what entity categories to return."
     },
-    "NamedEntityCategory": {
```

Comment:
The upcoming preview SDK (not this PR specifically) would be a good opportunity to remove other obsolete stuff, if there is any.

Reply:
I have a planned follow-up PR where I intend to remove NamedEntityCategory and `DocumentDb` which are now obsolete. I can tack on a commit to remove other "obvious" Obsolete types